### PR TITLE
Update ps-ref-update-package.md

### PR DESCRIPTION
**Pull request by xp44mm**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[5c83e3c](https://github.com/xp44mm/docs.microsoft.com-nuget/pull/1398/commits/5c83e3cc40c4e0af72b7ad9a0a9aee481b476296)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/tools/ps-ref-update-package.md](https://github.com/xp44mm/docs.microsoft.com-nuget/blob/patch-1/docs/tools/ps-ref-update-package.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/tools/ps-ref-update-package?branch=pr-en-us-1398) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C12%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201904120227170407-1398%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=TogrWX33d5d9KfY%2BUoQAFNRJerlCf08ZrBbOkjs98Bs%3D&st=2019-04-12T02%3A23%3A41Z&se=2019-05-13T02%3A28%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/tools/ps-ref-update-package.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Update-Package Microsoft.AspNet.MVC -Version 5.0.0.
 | Reinstall | Resintalls packages using their currently installed versions. See [Reinstalling and updating packages](../consume-packages/reinstalling-and-updating-packages.md). |
 | FileConflictAction | The action to take when asked to overwrite or ignore existing files referenced by the project. Possible values are *Overwrite, Ignore, None, OverwriteAll*, and *IgnoreAll* (3.0+). |
 | DependencyVersion | The version of the dependency packages to use, which can be one of the following:<br/><ul><li>*Lowest* (default): the lowest version</li><li>*HighestPatch*: the version with the lowest major, lowest minor, highest patch</li><li>*HighestMinor*: the version with the lowest major, highest minor, highest patch</li><li>*Highest* (default for Update-Package with no parameters): the highest version</li></ul>You can set the default value using the [`dependencyVersion`](../reference/nuget-config-file.md#config-section) setting in the `Nuget.Config` file. |
-| ToHighestPatch | Constrains upgrades to only versions with the same Minor version as the currently installed package. |
+| ToHighestPatch | equivalent to -Safe. |
```
Comment (karann-msft) [inline docs/tools/ps-ref-update-package.md]: that doesn't sound right. this should be a product issue if that is indeed the behavior. Can you point me to it or create one with a minimal repro.

Comment (xp44mm) [reply docs/tools/ps-ref-update-package.md]: in nuget console:
Update-Package -ToHighestPatch

从“nuget.org”检索包“Newtonsoft.Json 12.0.2” 
已从“packages.config”中删除程序包“Newtonsoft.Json 12.0.1”

Comment (xp44mm) [reply docs/tools/ps-ref-update-package.md]: reset the project and in nuget console:
Update-Package -safe

从“nuget.org”检索包“Newtonsoft.Json 12.0.2”
已从“packages.config”中删除程序包“Newtonsoft.Json 12.0.1”

Comment (zivkan) [reply docs/tools/ps-ref-update-package.md]: This is correct: 

https://github.com/NuGet/NuGet.Client/blob/314a51e00e0741beca7f2e6138afed0cc7191091/src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/UpdatePackageCommand.cs#L67-L69

Personally, I would prefer to this to be documented the other way around. Say that `-Safe` is an alias of `-ToHighestPatch`. `ToHighestPatch`'s current decsription is pretty bad though, `Safe`'s description is much better and strange that it's different.

Comment (karann-msft) [reply docs/tools/ps-ref-update-package.md]: @xp44mm - can you update the PR based on @zivkan's feedback. Thanks.

**Pull request merged**